app/test/test-support/src/main/java/io/syndesis/test/container/s2i/S2iProjectBuilder.java

```diff
@@ -48,16 +48,17 @@ public Project build(IntegrationSource source) {
                 .map(Objects::toString)
                 .orElse("s2i-assembly");
 
-        SyndesisS2iAssemblyContainer syndesisS2iAssemblyContainer = new SyndesisS2iAssemblyContainer(integrationName, projectDir, imageTag);
-        syndesisS2iAssemblyContainer.start();
+        final SyndesisS2iAssemblyContainer s2i = new SyndesisS2iAssemblyContainer(integrationName, projectDir, imageTag);
+        s2i.setCommand("sh", "-c", SyndesisS2iAssemblyContainer.S2I_ASSEMBLE_SCRIPT + " && sleep infinity");
```

Comment:
The container is setting the command already in https://github.com/syndesisio/syndesis/pull/8451/files#diff-870d232685fd3bdfd76497ab9f65ff39R50 I wonder if we can remove that or instead move the `sleep` inside the `CMD` statement. About the `sleep` statement I'm always scary to have something turning potentially in a zombie.

Reply:
This is about changing the command from `/usr/local/s2i/assemble` to `sh -c /usr/local/s2i/assemble && sleep infinity`, I don't want to change this for every use case of this container I do want to change it for this one. As discussed in the [description](https://github.com/syndesisio/syndesis/pull/8451#issue-416957215) the container is stopped by Testcontainers and it's removal is guaranteed by Testcontainers/ryuk.